lighthouse-core/audits/dobetterweb/dom-size.js

```diff
@@ -14,11 +14,33 @@
 
 const Audit = require('../audit');
 const Util = require('../../report/html/renderer/util.js');
+const i18n = require('../../lib/i18n');
 
 const MAX_DOM_NODES = 1500;
 const MAX_DOM_TREE_WIDTH = 60;
 const MAX_DOM_TREE_DEPTH = 32;
 
+const UIStrings = {
+  title: 'Avoids an excessive DOM size',
+  failureTitle: 'Uses an excessive DOM size',
+  description: 'Browser engineers recommend pages contain fewer than ' +
+    `~${MAX_DOM_NODES} DOM nodes. The sweet spot is a tree ` +
```

Comment:
plural. 
also can we get thousands separation on this number?

Reply:
aw that means we have to make it a template and all 😢

Reply:
how painful is that?

Reply:
well it expands its representation at every stage in the process for what seems like zero added value, making it a plural implies that the value could change but the description is static and could be translated just once.